modules/vsphere-problem-detector-metrics.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_vsphere/using-vsphere-problem-detector-operator.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator-metrics_{context}"]
+= Metrics for the vSphere Problem Detector Operator
+
+The {operator-name} exposes the following metrics for use by the {product-title} monitoring stack.
+
+.Metrics exposed by the {operator-name}
+[cols="2a,8a",options="header"]
+|===
+|Name |Description
+
+|`vsphere_cluster_check_total`
+|Cumulative number of cluster-level checks that the {operator-name} performed. This count includes both successes and failures.
+
+|`vsphere_cluster_check_errors`
+|Number of failed cluster-level checks that the {operator-name} performed. For example, a value of `1` indicates that one cluster-level check failed.
+
+|`vsphere_esxi_version_total`
+|Number of ESXi hosts with a specific version.
```

Comment:
@jsafrane, because of your comment for `CollectNodeESXiVersion` and after re-reading BZ#1915859, is the foll an improvement, or did I just poison it?

|`vsphere_esxi_version_total`
|Number of ESXi hosts with a specific version.
Be aware that if a host runs more than one node, the host is counted only once.

Reply:
> |vsphere_esxi_version_total
|Number of ESXi hosts with a specific version.
Be aware that if a host runs more than one node, the host is counted only once.


This is correct